src/sentry/api/endpoints/organization_member_invite/details.py

```diff
@@ -0,0 +1,62 @@
+from typing import Any
+
+from rest_framework.request import Request
+from rest_framework.response import Response
+
+from sentry.api.api_owners import ApiOwner
+from sentry.api.api_publish_status import ApiPublishStatus
+from sentry.api.bases.organization import OrganizationEndpoint
+from sentry.api.endpoints.organization_member.utils import RelaxedMemberPermission
+from sentry.api.exceptions import ResourceDoesNotExist
+from sentry.api.serializers import serialize
+from sentry.models.organization import Organization
+from sentry.models.organizationmemberinvite import OrganizationMemberInvite
+
+
+class OrganizationMemberInviteDetailsEndpoint(OrganizationEndpoint):
+    publish_status = {
+        "DELETE": ApiPublishStatus.EXPERIMENTAL,
+        "GET": ApiPublishStatus.EXPERIMENTAL,
+        "PUT": ApiPublishStatus.EXPERIMENTAL,
+    }
+    owner = ApiOwner.ENTERPRISE
```

Comment:
i think ecosystem / whatever the new name of the team is is owning auth, but can fix later